test/b2gedmntuples_cfg.py

```diff
@@ -654,14 +650,15 @@
 process.skimmedPatMET = cms.EDFilter(
     "PATMETSelector",
     #    src = cms.InputTag(metLabel, "", "PAT"),
-    src = cms.InputTag(metLabel, "", metProcess),
+#    src = cms.InputTag(metLabel, "", metProcess),
+    src = cms.InputTag(metLabel),
     cut = cms.string("")
     )
 
 process.skimmedPatPuppiMET = cms.EDFilter(
     "PATMETSelector",
     #    src = cms.InputTag(metLabel, "", "PAT"),
-    src = cms.InputTag(puppimetLabel, "", metProcess),
+    src = cms.InputTag(puppimetLabel),
```

Comment:
If you refer to the "metProcess" variable, it's needed later on as for taking the met filters one needs to differentiate between the case of data and MC where one takes either the RECO or the PAT process for the metUserData.